internal/controller/controllers/clusterdeployments_controller.go

```diff
@@ -108,6 +114,11 @@ func (r *ClusterDeploymentsReconciler) Reconcile(req ctrl.Request) (ctrl.Result,
 		return result, err
 	}
 
+	if *cluster.Status == models.ClusterStatusInstalled {
```

Comment:
Lets add another condition that will differentiate between day1 cluster and day2 cluster.
We probably don't need to update it on every reconcile, so probably need to check if it already set or not, and if we have day1 or day2 cluster in the backend. 
After this process is done we will want to delete day1 cluster and create day2 cluster using the same CRD (not for this PR)
Maybe adding a condition to the status that will mark this cluster as fully installed? @avishayt what do you think?

Reply:
I added condition and comment in code